lib/muffin_man/customer_feedback/v20240601.rb

```diff
@@ -0,0 +1,73 @@
+module MuffinMan
+  module CustomerFeedback
+    class V20240601 < SpApiClient
```

Comment:
Yeah, good thought. I'll make that change.

Reply:
Actually, I did have a conversation with the SPAPI guys, and this version is already being used live on seller central. Do you think it would be sufficient if I just added a comment in the class and also in the README.md file that notes it is in beta still? That way I don't have to mess with the naming convention for all the testing files as well?
@gavinritchie

Reply:
Yeah, if it's already live then I think it's probably fine to keep it as-is!